index.html

```diff
@@ -918,6 +937,37 @@ <h1 class="editableArea">Headline1</h1>
 						</div>
 					</div>
 				</li>
+
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M0,0 Q25,75 100 100" marker-start="url(#arrowstart)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M0,0 Q75,25 100 100" marker-end="url(#arrowend)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+				</li>
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M100,0 Q25,25 0 100" marker-start="url(#arrowstart)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M100,0 Q75,75 0 100" marker-end="url(#arrowend)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+				</li>
+				<svg class="hideme" xmlns="http://www.w3.org/2000/svg">
```

Comment:
would the change work without this hidden element? It is probably hard to reason about this, since it is hard to know where it belongs to and what it does.